### PR TITLE
redesigned NavBar

### PR DESCRIPTION
**Pull request by Suritam-Nandy**

### CODE REVIEW

File: landing.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Ritvick-me) [inline landing.html:6]: Change the title please.

File: landing.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="res/css/landing.css" />
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light">
+      <a class="navbar-brand" href="#"
+        ><img src="res/img/logos/logo-without-bg.png"
```
Comment (Ritvick-me) [inline landing.html]: `>') of anchor tag in the same line so it looks neat. Same for the img tag.

**Review by Ritvick-me — COMMENTED** [review]
The title of the PR should be in Present tense. Example Navbar redesign

File: landing.html (hunk 1 above)
Comment (Suritam-Nandy) [reply landing.html:6]: done

File: landing.html (hunk 2 above)
Comment (Suritam-Nandy) [reply landing.html]: done.

Comment (9267aakashsharma) [reply landing.html]: Use prettier to pretify your code.

**Pull request merged**